tartiflette_aiohttp/__init__.py

```diff
@@ -158,11 +158,6 @@ def register_graphql_handlers(
     if context_factory is None:
         context_factory = default_context_factory
```

Comment:
Also can you transform the "default_context_factory" into an asynccontextmanager ? so default is the nominal use case.

Reply:
Done!

Reply:
Yep, perfect !!